docs/source/markdown/podman-image-diff.1.md

```diff
@@ -40,7 +55,9 @@ A /usr/local/bin/docker-entrypoint.sh
 ```
 
 ## SEE ALSO
-podman(1)
+
+**[podman(1)](podman.1.md)**
```

Comment:
NEAT!   I didn't think about this, we ought to add this as a requirement in the manpage syntax if not there already.